EventFilter/EcalRawToDigi/plugins/alpaka/UnpackPortable.h

```diff
@@ -0,0 +1,22 @@
+#ifndef EventFilter_EcalRawToDigi_plugins_alpaka_UnpackPortable_h
+#define EventFilter_EcalRawToDigi_plugins_alpaka_UnpackPortable_h
+
+#include "CondFormats/EcalObjects/interface/alpaka/EcalElectronicsMappingDevice.h"
+#include "DataFormats/EcalDigi/interface/alpaka/EcalDigiDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "DeclsForKernels.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::ecal::raw {
+
+  void unpackRaw(Queue&,
+                 InputDataHost const&,
+                 EcalDigiDeviceCollection&,
+                 EcalDigiDeviceCollection&,
+                 EcalElectronicsMappingDevice const&,
+                 uint32_t const,
+                 uint32_t const);
```

Comment:
could you add here the name of the parameters ?
 ```suggestion
   void unpackRaw(Queue& queue,
                  InputDataHost const& inputHost,
                  EcalDigiDeviceCollection& digisDevEB,
                  EcalDigiDeviceCollection& digisDevEE,
                  EcalElectronicsMappingDevice const& mapping,
                  uint32_t const nfedsWithData,
                  uint32_t const nbytesTotal);
```